utils/market_makers.py

```diff
@@ -28,16 +34,81 @@ def calculate_y(self, x):
   def calculate_spot_price(self):
     return 1
 
+  def define_sell_buy(self,typeTokenSell,balanceX,balanceY):
+    if typeTokenSell == 'X':
+      tokensData = {
+        'typeTokenBuy': 'Y',
+        'initialAmountSell': balanceX,
+        'initialAmountBuy': balanceY,
+      }
+      return tokensData
+    if typeTokenSell == 'Y': 
+      tokensData = {
+        'typeTokenBuy': 'X',
+        'initialAmountSell': balanceY,
+        'initialAmountBuy': balanceX,
+      }
+      return tokensData
+
+  def calculate_trade(self, initialAmountSell, initialAmountBuy, amountTokenSell):
+    k =  initialAmountSell + initialAmountBuy
+    amountTokenBuy = initialAmountSell + initialAmountBuy + amountTokenSell - k
+    price = amountTokenBuy/amountTokenSell
+    finalAmountSell = initialAmountSell+amountTokenSell
+    finalAmountBuy = initialAmountBuy-amountTokenBuy
+
+    transaction = {
+      'amountTokenBuy': amountTokenBuy,
+      'price': price,
+      'transactionSell': [initialAmountSell,finalAmountSell],
+      'transactionBuy': [initialAmountBuy,finalAmountBuy],
+      'label': ['Before the trade', 'After the trade']
+    }
+
+    return transaction
+
 
 class Uniswap(MarketMaker):
   def get_constant(self, x, y):
-    return x+y
+    return x*y
   
   def calculate_y(self, x):
     return self.constant / x
   
   def calculate_spot_price(self, x):
     return self.constant / (x**2)
+  
+  def define_sell_buy(self,typeTokenSell,balanceX,balanceY):
```

Comment:
Usually, variable and function names are with under-case letters separated by an underscore. The format that you used is used for classes. Also, separate the variables with a space.

Reply:
done [096028bcaf5e95e7b89df98b91277ddd389aab23]( https://github.com/luizakp/streamlittest/pull/2/commits/096028bcaf5e95e7b89df98b91277ddd389aab23)